.github/workflows/ci.yml

```diff
@@ -38,23 +42,19 @@ jobs:
           workspaces: src/Temporalio/Bridge
 
       - name: Setup .NET
-        uses: actions/setup-dotnet@v3
+        uses: actions/setup-dotnet@v4
+        with:
+          # Specific .NET version required because GitHub macos ARM image has
+          # bad pre-installed .NET version
+          dotnet-version: 6.x
 
-      - name: Install protoc (x64)
-        # Does not work on arm
-        if: ${{ matrix.os != 'ubuntu-arm' }}
-        uses: arduino/setup-protoc@v1
+      - name: Install protoc
+        uses: arduino/setup-protoc@v3
         with:
-          # TODO(cretz): Upgrade when https://github.com/arduino/setup-protoc/issues/33 fixed
-          version: '3.x'
+          # TODO(cretz): Can upgrade proto when https://github.com/arduino/setup-protoc/issues/99 fixed
```

Comment:
Not very important, but I'm a bit puzzled by this TODO. Looks like this is about the _default_ version of protoc that `setup-protoc` will install. But if you specify the version number explicitly, there should be no problem.

Reply:
> Looks like this is about the default version of protoc that setup-protoc will install.

We want to pin our protoc version (different versions can generate different code). This is just saying we can't upgrade past 23.x because of that issue.